cmd/detect-latest-release/main.go

```diff
@@ -7,8 +7,8 @@ import (
 	"log"
 	"os"
 
-	"github.com/creativeprojects/go-selfupdate"
```

Comment:
Can you please fix all the paths back to the original ones? as it stands it's all broken when using the paths from your fork.
Thanks 👍🏻